react-frontend/src/reducers/appReducer.js

```diff
@@ -1,4 +1,6 @@
-const defaultState = {};
+const defaultState = {
+  token: "",
```

Comment:
Make default value `null`. Doing so means developers ensure that code handles null values.